src/core_plugins/kibana/public/management/sections/settings/advanced_row.js

```diff
@@ -65,11 +65,11 @@ uiModules.get('apps/management')
         });
       };
 
-      $scope.isValueNonDefault = (conf) => {
-        return !conf.isCustom
-          && conf.value !== undefined
-          && conf.value !== ''
-          && String(conf.value) !== String(conf.defVal);
+      $scope.isDefaultValue = (conf) => {
+        return conf.isCustom
```

Comment:
Oh, I see, that's a customer *setting*, not a custom *value*. Logically the code is OK, I just thought the value meant something else. A comment here about what isCustom actually means would probably be helpful then.